File: ClientApp/src/app/models/leave/leave-apply.model.ts
import { DropDownList } from "../common/dropdown";

export class LeaveApply {
    public id: string;
    public leaveTypeId: string;
    public startDate: Date;
    public endDate: Date;
    public noOfDays: string;
    public reasonForApply: string;
    public leaveType:DropDownList[];
    public userId : string;
}

File: A4.Empower-Git/A4.Empower/A4.BAL/Leave/LeaveApplyModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace A4.BAL
{
    public class LeaveApplyModel
    {
        [Required]
        public DateTime StartDate { get; set; }
        [Required]
        public DateTime EndDate { get; set; }
        [Required]
        public string LeaveTypeId { get; set; }
        [Required]
        public string ReasonForApply { get; set; }

        [Required]
        public string UserId { get; set; }




    }
}
